jest.config.js

```diff
@@ -26,20 +26,14 @@
 process.env.TZ = 'UTC'
 
 module.exports = {
-  // roots: ['<rootDir>'],
-  // testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.tsx?$',
   automock: false,
   moduleDirectories: ['./node_modules', './packages'],
   moduleFileExtensions: ['js', 'jsx', 'ts', 'tsx', 'json', 'node'],
   moduleNameMapper: {
-    '\\.(css)$': '<rootDir>/config/jest/styleMock.js',
+    '@looker/(.+)$': '<rootDir>/packages/$1/src',
```

Comment:
This allows jest to resolve packages however package directories MUST match package names for this to work. Hence renaming of several packages within this PR.